draft-ietf-quic-transport.md

```diff
@@ -1207,41 +1208,53 @@ be routed back to the endpoint and identified by the endpoint upon receipt.
 
 A zero-length connection ID MAY be used when the connection ID is not needed for
 routing and the address/port tuple of packets is sufficient to associate them to
-a connection. An endpoint whose peer has selected a zero-length connection ID
+a connection.  An endpoint whose peer has selected a zero-length connection ID
 MUST continue to use a zero-length connection ID for the lifetime of the
 connection and MUST NOT send packets from any other local address.
 
-When an endpoint has requested a non-zero-length connection ID, it will issue a
-series of connection IDs over the lifetime of a connection. The series of
-connection IDs issued by an endpoint is ordered, with the final connection ID
-selected during the handshake coming first.  Additional connection IDs are
-provided using the NEW_CONNECTION_ID frame ({{frame-new-connection-id}}), each
-with a specified sequence number.  The series of connection IDs issued SHOULD be
-contiguous, but might not appear to be upon receipt due to reordering or loss.
-
-Each connection ID MUST be used on only one local address. When packets are sent
-for the first time on a new local address, a new connection ID MUST be used with
-a higher sequence number than any connection ID previously used on any local
-address.  At any time, an endpoint MAY change to a new connection ID on a local
-address already in use.
-
-An endpoint MUST NOT send packets with a connection ID which has a lower
-sequence number than the highest sequence number of any connection ID ever sent
-or received on that local address.  This ensures that when an endpoint migrates
+An endpoint whose peer has selected a non-zero-length connection ID will
+potentially use multiple connection IDs over the lifetime of a connection.
```

Comment:
"Using" multiple connection ID sounds confusing—we have two ways to use CIDs, either to receive on or to send to. In this case, it's only about receiving on. So perhaps "will potentially generate multiple connection IDs" or "will potentially accept packets for multiple connection IDs"

Reply:
Good call, reworded